### PR TITLE
fix(FeatureCard): margin css fix

### PR DESCRIPTION
**Pull request by greeshmareddyg**
### Related Ticket(s)

https://github.com/carbon-design-system/ibm-dotcom-library/issues/2550

### Changelog

**Changed**

-Extra 16px margin in medium

### CODE REVIEW

File: packages/styles/scss/components/feature-card/_feature-card.scss [hunk 1]
```diff
@@ -22,16 +22,6 @@
...
 
-    @include carbon--breakpoint('md') {
-      margin-right: $spacing-05;
-      margin-left: $spacing-05;
-    }
-
-    @include carbon--breakpoint-down('md') {
-      margin-right: -$spacing-05;
-      margin-left: -$spacing-05;
-    }
```
Comment (RaphaelAmadeu) [inline packages/styles/scss/components/feature-card/_feature-card.scss:33]: These lines are changing `small` and not `medium` layout.

**Comment by greeshmareddyg** [comment]
> @greeshmareddyg this PR is getting a bit stale, can you take a look at @RaphaelAmadeu 's feedback?

yeah, addressed it @jeffchew

File: packages/styles/scss/components/feature-card/_feature-card.scss (hunk 1 above)
Comment (greeshmareddyg) [reply packages/styles/scss/components/feature-card/_feature-card.scss]: missed it, just fixed @RaphaelAmadeu , thank you.

**Comment by greeshmareddyg** [comment]
> @greeshmareddyg It looks like the margins are off in the smallest breakpoint. The card should have negative left/right margins and be 100% wide.
> 
> image

@kennylam I've reverted for below medium, its occupying the fullspace now for small devices. could you check now.

image

**Comment by annawen1** [comment]
@RaphaelAmadeu is this good to go?

**Pull request merged**